src/_playground/App.scss

```diff
@@ -96,12 +95,28 @@ $library-color: #61dafb;
             }
         }
 
+        @media (max-width: $mediaMaxWidth) {
+            .frDocs-Sidebar.sidebar-hidden{
+                        margin-left: -20rem;
```

Comment:
Are we sure that's wide enough?  The sidebar has its width specified in pixels so this should probably also be in pixels.